### PR TITLE
Add link to Unix permissions illustration

### PR DESCRIPTION
**Pull request by wrekone**
Added link per https://github.com/enkidevs/curriculum/issues/750

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md: 
 > unexpected end of the stream within a single quoted scalar at line 32, column 1:
    
    ^

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md?ref=e335c78825b6064b2f08c2c300819823bb0b4966)

**Comment by kapnobatai137** [comment]
Thank you @wrekone.

File: linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ links:
 
   - '[linuxcommand.org](http://linuxcommand.org/lts0070.php){website}'
   - '[www.yolinux.com](http://www.yolinux.com/TUTORIALS/LinuxTutorialManagingGroups.html){website}'
```
Comment (kapnobatai137) [inline linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md]:
```suggestion
  - '[Managing Group Access](http://www.yolinux.com/TUTORIALS/LinuxTutorialManagingGroups.html){website}'
```

File: linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ links:
 
   - '[linuxcommand.org](http://linuxcommand.org/lts0070.php){website}'
```
Comment (kapnobatai137) [inline linux/user-and-file-management/file-permissions/set-file-permissions-for-users.md]: the link seems unresponsive:
image


```suggestion
```

**Pull request merged**